src/Components/OptionCardGroup/OptionCardGroup.css

```diff
@@ -49,7 +49,7 @@
   margin-bottom: 1.5rem;
 }
 
-@media (max-width: 420px) {
+@media (max-width: 610px) {
```

Comment:
Did you test with 211? FYI, you can use `rem` instead of `px` if you want the media query to be relative to font size.

Reply:
Yes, I have a demo of the media query on our default and 211 sites.